tensor_comprehensions/pybinds/tclib.cc

```diff
@@ -459,7 +476,13 @@ PYBIND11_MODULE(tclib, m) {
           "unchecked_run",
           &TcExecutor::uncheckedRun,
           py::arg("inputs"),
+          py::arg("outputs") = py::tuple())
+      .def(
+          "profile_kernel",
```

Comment:
Please call this "profile" too.

Reply:
Nicolas Vasilache prefers "profile_kernel" as it doesnt bind the whole ProfileInfo structure.

Reply:
> as it doesnt bind the whole ProfileInfo structure

why not (mention that in the commit message)?  

Anyway, there are two points:
- the C++ function should have the same name as the python function, otherwise we will quickly loose grip of what corresponds to what and risk creating name conflicts;
- I strongly prefer python API to be as close to C++ as possible;  we went the other way, you can ask @nicolasvasilache how much time did he spend reworking that to remove overhead and inconsistencies.

Ok, the function only returns the kernel time.  It is sort of indicated in the name, but not immediately clear (one can interpret it as "profile one kernel" as opposed to "profile the entire network" or even "profile the compilation and the kernel").  I don't see another name for now, but this is something that should be mentioned in the commit message.